### ISSUE TITLE
C :heart: ding music

### ISSUE THREAD

**Issue by xaca** [xaca/coding-music#23]
Looking to songs recomendations in diferent genres 🎵 🎵 ❤️

**Comment by Gatoze** [comment]
What about Lo-fi Hip Hop it´s very relaxing and good to keep your mind in a single thing

**Comment by palvisandy** [comment]
i want to see your xcode for that

**Comment by xaca** [comment]
@Gatoze All genre all welcome, go ahead!

**Comment by I-Go-By-M** [comment]
So, just random songs in different genres?

Melvins - Hooch
Dizzy Gillepsie - Salt Peanuts
Travis Scott - Highest in the Room
Boretta - Honey Heart

**Comment by xaca** [comment]
@I-Go-By-M Yes, take a look on [readme](https://github.com/xaca/coding-music) We have a lot of folders, each user, order the songs depend on the genre.

**Comment by Cheesehacker** [comment]
folk punk
Dazed and Days- Flea bitten drifters
Johny Hobo- Acid song

**Comment by I-Go-By-M** [comment]
Fixed vers.

I propose a Hard Rock/Heavy Metal genre. Don't Stop Me Now is too different from Enter Sandman. You already have a section for Indie music, so it would only make sense. Also, a genre for Bossa Nova pls.

[Melvins - Hooch](https://www.youtube.com/watch?v=gRTrAI3Yetk) (Hard Rock)
[Dizzy Gillepsie - Salt Peanuts](https://www.youtube.com/watch?v=y6N8eL-cuEU) (English Jazz)
[Travis Scott - Highest in the Room](https://www.youtube.com/watch?v=tfSS1e3kYeo) (English Rap)
[Boreta - Honey Heart](https://www.youtube.com/watch?v=IcXS7Q4k728) (Electro)
[Laurindo Almeida - The Lamp is Low](https://www.youtube.com/watch?v=lkk6m14htzw) (Bossa Nova)

**Comment by xaca** [comment]
@Cheesehacker & @I-Go-By-M  Thank you for your recomendations, remember update readme file on each genre folder. Thank you!
